packages/url_launcher/url_launcher/test/url_launcher_test.dart

```diff
@@ -44,7 +44,7 @@ void main() {
     });
 
     test('default behavior', () async {
-      await launch('http://flutter.dev/');
+      await launch('http://flutter.dev/', windowName: "_blank");
```

Comment:
Passing windowName is *not* the default behavior, please undo this test. Ideally you should add another test of what happens when you pass `windowName`.